client/css/ProductOverviewCSS.scss

```diff
@@ -0,0 +1,94 @@
+.productOverviewContainer {
+  display: flex;
+}
+
+.productInfo{
+  border: 2px solid black;
+  width: 50%;
+  margin: 0 1%;
+}
+
+.imageGalleryContainer {
+  display: flex;
+  height: 1000px;
+  width: 100%;
+  align-items: center;
+}
+
+.mainImageContainer {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  border: 2px solid black;
+  justify-content: space-around;
+  align-items: center;
+  order: 3;
+};
+
+
+.mainImage {
+  width: 50%;
+  min-height: 50%;
+  max-height: 50%;
+}
+
+.productDescription {
+  border: 2px solid black;
+  margin: 1% 0;
```

Comment:
I cringe a little every time I see these percentages. I think a set value would be better and then a media query for different screen sizes would be best.

Reply:
I think we should discuss sizing as a group. Not the most important right now for MVP. Media queries are a good idea, not sure on the best practices of implementing them though.